R/build_GRAN_all.R

```diff
@@ -17,7 +17,7 @@ build_GRAN_all <- function(versions){
 	  orig.version = dir_version()
 		for (version in versions){
 		  set_version(version)
-			system("Rscript -e 'library(granbuild);dl_build_src();dl_build_bin()'")
```

Comment:
This is the source of @jread-usgs 's complaint. Unecessary to dl_build_src for every R version. Just needs to be done once, then dl_build_bin() for all versions.